actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java

```diff
@@ -831,6 +862,10 @@ public Response apply(String parameter) {
     }
     requestMap.put(JsonKey.PASSWORD, userMap.get(JsonKey.PASSWORD));
     if (StringUtils.isNotBlank(callerId)) {
+      if (StringUtils.isNotEmpty(managedBy)) {
+        requestMap.put(JsonKey.EMAIL, managedByEmail);
```

Comment:
this is used to send notification about account creation. But seems like the map is carrying this as if the managed user has it. Dislike this idea.